README.txt

```diff
@@ -2,26 +2,26 @@
-│                                                        ...... ...... ...... ...... ...... │
-│                                          ...... ...... ...... ...... ...... ...... ...... │
-│                                          ...... ...... ...... ...... ...... ...... ...... │
-│                            ...... ...... ...... ...... ...... ...... ...... ...... ...... │
-│              ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... │
-│...... ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... │
-│...... ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... │
-│..22.. ..25.. ..26.. ..32.. ..35.. ..36.. ..49.. ..51.. ..52.. ..54.. ..89.. .100.. .109.. │
-│Lua    Go     Erlang C      Python Ruby   JS     Java   Rust   Dart   Swift  C#     C++    │
-└───────────────────────────────────────────────────────────────────────────────────────────┘
+┌──────────────────────────────────────────────────────────────────────────────────────────────────┐
+│                                                                                           ...... │
+│                                                                                           ...... │
+│                                                                                           ...... │
+│                                                                                           ...... │
+│                                                                                           ...... │
+│                                                                                           ...... │
+│                                                                                           ...... │
+│                                                                                           ...... │
+│                                                                                           ...... │
+│                                                                                           ...... │
+│                                                                                           ...... │
+│                                                                                           ...... │
+│                                                                                           ...... │
+│                                                                             ...... ...... ...... │
+│                                                                      ...... ...... ...... ...... │
+│                                                 ...... ...... ...... ...... ...... ...... ...... │
+│  22   ..25.. ..26.. ..32.. ..35.. ..36.. ..49.. ..51.. ..52.. ..54.. ..89.. .100.. .109.. .417.. │
+│Lua    Go     Erlang C      Python Ruby   JS     Java   Rust   Dart   Swift  C#     C++    COBOL  │
+└──────────────────────────────────────────────────────────────────────────────────────────────────┘
```

Comment:
It's a shame adding this flattens half of the graph so we can't really see the difference anymore. It shifts the graph from being a comparison of languages to being a comparison of everything against COBOL which wasn't the original intent.

Even increasing the height of the graph much doesn't help much 😞 .

```
┌──────────────────────────────────────────────────────────────────────────────────────────────────┐
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                           ...... │
│                                                                                    ...... ...... │
│                                                                             ...... ...... ...... │
│                                                                      ...... ...... ...... ...... │
│                                                                      ...... ...... ...... ...... │
│                                                                      ...... ...... ...... ...... │
│                                                                      ...... ...... ...... ...... │
│                                                                      ...... ...... ...... ...... │
│                                                        ...... ...... ...... ...... ...... ...... │
│                                          ...... ...... ...... ...... ...... ...... ...... ...... │
│                                          ...... ...... ...... ...... ...... ...... ...... ...... │
│                     ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... │
│...... ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... │
│...... ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... ...... │
│..22.. ..25.. ..26.. ..32.. ..35.. ..36.. ..49.. ..51.. ..52.. ..54.. ..89.. .100.. .109.. .417.. │
│Lua    Go     Erlang C      Python Ruby   JS     Java   Rust   Dart   Swift  C#     C++    COBOL  │
└──────────────────────────────────────────────────────────────────────────────────────────────────┘
```

Reply:
I know, maybe it doesn't need to be in graph. 
Just wanted to show definite winner in count of keywords.

Reply:
Yeah, maybe I can keep it in the graph, and just manually put it in so that it is capped a height not as high but with the number still saying `417`. I think it's worth including.